trunk/NDHMS/Routing/module_NWM_io_dict.F

```diff
@@ -17,7 +17,8 @@ module module_NWM_io_dict
 implicit none
 
 ! Declare parameter values for module.
-integer, parameter :: numChVars = 10
+!character (len=512), parameter :: codeVersion = "WRF-Hydro v5.0.0-dev"
```

Comment:
Old version of model (NWM v1.2) had WRF-Hydro version. I am not sure if this is needed.

Reply:
remove